HACKING.md

```diff
@@ -83,12 +83,26 @@ you can push the images to OpenShift's Docker repo like this:
 
         DOCKER_REGISTRY=172.30.1.1:5000 DOCKER_ORG=`oc project -q` make all
         
-4. Finally, when creating the new app from the template you need to
-   specify the Docker repo, otherwise OpenShift will still try to pull 
-   the image from docker.io:
-   
-        oc new-app strimzi-ephemeral -p IMAGE_REPO_NAME=172.30.1.1:5000/myproject
+4. In order to use the just built Cluster Operator image, you need to update the `image` field in the `examples/install/cluster-operator/07-deployment.yml` with the new value `172.30.1.1:5000/myproject/cluster-operator:latest`.
```

Comment:
So, why not save one step and ask the user to update all the images in the `deployment.yaml`?

Reply:
Yeah right. Done.